### DISCUSSION TITLE
Can I use filters from files?

### DISCUSSION THREAD

**Discussion by vmvartiainen** [Burnett01/rsync-deployments#66] [General]
Hey!

I was just wondering if it's possible to use flag --exlude-from? It needs a file that includes filter rules. Is this possible?

**Comment by Burnett01** [comment]
Hi @vmvartiainen that should be possible. There is no restriction on what rsync flags you can use.

**Reply by vmvartiainen** [reply]
Thanks. Actually I tested it succesfully. It's working perfectly!

**Comment by Burnett01** [comment]
You can check these lunks for examples

https://github.com/Burnett01/rsync-deployments/issues/21#issuecomment-979474255

https://github.com/Burnett01/rsync-deployments/issues/20#issuecomment-969949542
